### PR TITLE
Add Scopeset Expander

### LINKED ISSUE #40: Scopeset expansions
This thing https://tools.taskcluster.net/auth/scopes/expansions

### PR DESCRIPTION
**Pull request by helfi92**
Resolves #40.

### CODE REVIEW

File: src/views/Scopes/index.jsx [hunk 1]
```diff
@@ -5,10 +5,10 @@ import RouteWithProps from '../../components/RouteWithProps';
 import loadable from '../../utils/loadable';
 
 const ViewScopes = loadable(() =>
-  import(/* webpackChunkName: 'Clients.ViewScopes' */ './ViewScopes')
+  import(/* webpackChunkName: 'Scopes.ViewScopes' */ './ViewScopes')
```
Comment (eliperelman) [inline src/views/Scopes/index.jsx]: Rename `ViewScopes` to `ListScopes`.

**Comment by helfi92** [comment]
screen shot 2018-07-03 at 4 06 25 pm

**Comment by eliperelman** [comment]
Nevermind, you did lol.

File: src/views/ScopesetExpander/index.jsx [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+    ];
+  }
+
+  render() {
+    const { classes, user, onSignIn, onSignOut } = this.props;
+    const { scopes, scopeText } = this.state;
+
+    return (
+      <Dashboard
+        title="Scopeset Expander"
```
Comment (eliperelman) [inline src/views/ScopesetExpander/index.jsx]: Rename this to "Expand Scopesets". We want to name these in action style rather than tool style.

**Pull request merged**